Tests/MessagingInApp/MessagingInAppIntegrationTests.swift

```diff
@@ -264,6 +264,7 @@ extension MessagingInAppIntegrationTest {
     }
 
     func onDoneFetching(messages: [Message]) {
+        // swiftlint:disable:next force_cast
         (Gist.shared.messageQueueManager as! MessageQueueManagerImpl).processFetchedMessages(messages)
```

Comment:
I prefer to not have private functions such as `processFetchedMessages` as part of the `MessageQueueManager` protocol as the protocol is meant to be the `public` API that the SDK calls and only the tests uses `processFetchedMessages`. 

Instead of directly calling `processFetchedMessages`, we could mock the `GistQueueNetwork` HTTP request which then calls `processFetchedMessages`. But I believe that change would go beyond the scope of this PR and may not be worth the effort when this call does the same thing.